compiler/luci/service/src/CircleShapeInferenceRule.cpp

```diff
@@ -235,6 +235,11 @@ loco::NodeShape use_paddings(const CIRCLENODE *node, const luci::CircleConst *pa
   output_shape.rank(input_shape.rank());
   for (int32_t ni = 0; ni < n; ++ni)
   {
+    if (not input_shape.dim(ni).known())
+    {
+      output_shape.dim(ni).unset();
+      continue;
+    }
```

Comment:
I think these changes are not needed as `use_paddings()` in `HelperPads.h` will be used.
Am I correct?

Reply:
You are right.
This part seems to be unnecessary in the current commit.
Other pad ops , padV2 and mirrorPad,  will use `use_paddings()` in `HelperPads.h` in the future, so this code is unnecessary.